OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -573,6 +573,12 @@ static bool Kick(S server, Connection conn, Session.Client client, string s)
 				return true;
 			}
 
+			if (server.State == ServerState.GameStarted && !client.IsObserver)
```

Comment:
I think this is checking whether the admin is an observer. This should be using the `kickClientID` or `kickConn` instead.

Reply:
🤦‍♂